spock-specs/src/test/groovy/org/spockframework/mock/runtime/mockito/MockitoStaticMocksSpec.groovy

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.mock.runtime.mockito
+
+import org.mockito.Answers
+import org.mockito.Mockito
+import org.spockframework.mock.CannotCreateMockException
+import org.spockframework.mock.IDefaultResponse
+import org.spockframework.mock.IMockInvocation
+import org.spockframework.runtime.InvalidSpecException
+import spock.lang.Specification
+import spock.mock.IStaticMock
+import spock.mock.MockMakers
+
+@SuppressWarnings(["GroovyAssignabilityCheck", "UnnecessaryQualifiedReference"])
+class MockitoStaticMocksSpec extends Specification {
+  private static final String REAL_VALUE = "RealValue"
+  private static final String MOCK_VALUE = "MockValue"
+
+  def "Mockito API Test static mock with response"() {
```

Comment:
❓ why these Mockito API tests? Just to make it easy if there is a problem with Mockito and not the Spock integration of it?

🤔 Maybe you should also add one that mixes both usages.

Reply:
Yes I want to ensure, that the static features are working without the spock interaction, if these are failing the problem comes from Mockito, not our integration. This woul help us in the future to investigate issues more quickly.

I will add tests, which will try both usages together.